appium-tests/common/common.spec.js

```diff
@@ -114,14 +116,15 @@ describe('Contacts Android', function () {
             })
             .click()
             .context(webviewContext)
-            .executeAsync(function (pID, cb) {
+            .executeAsync(function (pID, getOnlyBirthday, cb) {
```

Comment:
Okay, that's fine then. It might be better to just always return a contact object but convert the birthday to a string so that the function always returns the same type.